// SPDX-FileCopyrightText: 2025 Carnegie Mellon University - Living Edge Lab
//
// SPDX-License-Identifier: GPL-2.0-only

syntax = "proto3";
package protocol.log;

/*
 * Types of general log messages
 */
message InfoLog {
  string msg = 1;
}

message DebugLog {
  string msg = 1;
}

message WarningLog {
  string msg = 1;
}

message ErrorLog {
  string msg = 1;
}
